CMS-Enterprise/mint-app
# Noref/fix solution types

# NOREF

## Description

- Updates SQL types and migrations to reference `CONTRACTOR` as a SolutionType instead of `CONTRACT` (which appeared to just be a typo/mismatch from SQL to GQL)

## How to test this change

1.  Ensure that adding a custom solution through postman with the `CONTRACTOR` type works as expected

### Before
Below is a screenshot of the failure you'll see outside this branch, courtesy of @garyjzhao 
![image](https://github.com/user-attachments/assets/ff6952ba-89c0-4a8e-8d7d-e16efeabe1f5)

### After
![image](https://github.com/user-attachments/assets/5e9b9bdb-8c7f-4c81-b30b-1f42e458b561)

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
